Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -2058,6 +2058,28 @@ int PhysicalParticleContainer::GetRefineFac(const Real x, const Real y, const Re
     return ref_fac;
 }
 
+/*Function to store values for back-tranformed diagnostic"*/
+void
+PhysicalParticleContainer::warpx_copy_attribs(const long* np,
+								const amrex::Real* xp, const amrex::Real* yp, const amrex::Real* zp,
```

Comment:
Could you please fix indentation and remove tabs?

Reply:
Of course! Sorry